app/modules/collaborations/models.py

```diff
@@ -14,11 +15,20 @@
 
 
 class CollaborationUserState:
-    ALLOWED_STATES = ['declined', 'approved', 'pending', 'not_initiated']
+    ALLOWED_STATES = [
+        'declined',
+        'approved',
+        'pending',
+        'not_initiated',
+        'revoked',
+        'creator',
+    ]
     DECLINED = ALLOWED_STATES[0]
     APPROVED = ALLOWED_STATES[1]
     PENDING = ALLOWED_STATES[2]
     NOT_INITIATED = ALLOWED_STATES[3]
+    REVOKED = ALLOWED_STATES[4]
+    CREATOR = ALLOWED_STATES[5]
```

Comment:
not happy with "Creator" as a state, INACTIVE, OBSERVER?

Reply:
What about MANAGER?

Reply:
Manager is a role, like creator, whereas this is supposed to be a state..

Reply:
Right, but since it's a state for a CollaborationUserAssociation for a user that's not actively a part of the data sharing I figured that would make sense. The only user on a collaboration that wasn't actually part of the data sharing would be a user_manager that created one for others. I'm not too attached to the name for this state but that was my thinking.